src/components/RoundedBackground.tsx

```diff
@@ -1,13 +1,13 @@
 import { heightPercentageToDP as hp } from 'react-native-responsive-screen'
 import styled from 'styled-components/native'
 
-const RoundedBackground = styled.View<{ color?: string; height?: string }>`
+const RoundedBackground = styled.View<{ color?: string; height?: string; marginBottom?: string }>`
   width: 140%;
   height: ${props => props.height ?? '50%'};
   background-color: ${props => props.color ?? props.theme.colors.primary};
   border-bottom-left-radius: ${hp('60%')}px;
   border-bottom-right-radius: ${hp('60%')}px;
-  margin-bottom: ${props => props.theme.spacings.xxl};
+  margin-bottom: ${props => props.marginBottom ?? props.theme.spacings.xxl};
```

Comment:
Why are you adding this? It's not used anywhere? If it is from previous code changes that don't apply anymore, then please revert it :)

Reply:
I removed it. Seems like it did not change at all. But I found out what was the problem and now resolving it. Thank you, Leandra!